webpack.config.renderer.dev.js

```diff
@@ -234,6 +237,31 @@ export default merge.smart(baseConfig, {
 
     new ExtractTextPlugin({
       filename: '[name].css'
+    }),
+
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, 'app', 'app.html')
+    }),
+
+    new AddAssetHtmlPlugin({
+      filepath: path.resolve(__dirname, 'dll', 'renderer.dev.dll.js'),
+      includeSourcemap: false
+    }),
+
+    new CspHtmlWebpackPlugin({
+      'default-src': "'self'",
+      'object-src': "'none'",
+      'connect-src': [
+        "'self'",
+        'http://localhost:*',
+        'ws://localhost:*',
+        'https://api.coinmarketcap.com',
+        'https://zap.jackmallers.com',
+        'https://testnet-api.smartbit.com.au'
+      ],
+      'script-src': ["'self'", 'http://localhost:*', "'unsafe-eval'"],
+      'font-src': ["'self'", 'data:', 'http://localhost:*', 'https://fonts.googleapis.com', 'https://s3.amazonaws.com', 'https://fonts.gstatic.com'],
+      'style-src': ["'self'", 'blob:', 'https://fonts.googleapis.com', 'https://s3.amazonaws.com', 'https://fonts.gstatic.com', "'unsafe-inline'"]
```

Comment:
Is it possible to remove `unsafe-inline` now?

Reply:
No. Without it we get errors like this:

> setInnerHTML.js:44 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' blob: https://fonts.googleapis.com https://s3.amazonaws.com https://fonts.gstatic.com". Either the 'unsafe-inline' keyword, a hash ('sha256-E3dZVGA62Z1nIA/nziyL8qEhz+nm68ehiFEWHvx0rnc='), or a nonce ('nonce-...') is required to enable inline execution.